src/PhpSpreadsheet/Reader/Html.php

```diff
@@ -283,15 +284,34 @@ protected function releaseTableStartColumn(): string
      * @param int|string $row
      * @param mixed $cellContent
      */
-    protected function flushCell(Worksheet $sheet, $column, $row, &$cellContent): void
+    protected function flushCell(Worksheet $sheet, $column, $row, &$cellContent, array $attributeArray): void
     {
         if (is_string($cellContent)) {
             //    Simple String content
             if (trim($cellContent) > '') {
                 //    Only actually write it if there's content in the string
                 //    Write to worksheet to be done here...
-                //    ... we return the cell so we can mess about with styles more easily
-                $sheet->setCellValue($column . $row, $cellContent);
+                //    ... we return the cell, so we can mess about with styles more easily
+
+                // Set cell value explicitly if there is data-type attribute
+                if (isset($attributeArray['data-type'])) {
+                    $datatype = $attributeArray['data-type'];
+                    if ($datatype == DataType::TYPE_STRING || $datatype == DataType::TYPE_STRING2) {
+                        if (substr($cellContent, 0, 1) === '=' || is_numeric($cellContent)) {
```

Comment:
I do not understand the purpose of the is_numeric test. Explicitly setting a numeric value to type string should not require quotePrefix. If your testing indicated that it did require it, please add a test to show why.